metron-platform/metron-common/README.md

```diff
@@ -167,6 +172,14 @@ The `!=` operator is the negation of the above.
 | [ `WEEK_OF_YEAR`](#week_of_year)                                                                   |
 | [ `YEAR`](#year)                                                                                   |
 
+### `APPEND_IF_MISSING`
+  * Description: Appends the suffix to the end of the string if the string does not already end with any of the suffixes.
+  * Input:
+    * string - The string to be appended.
+    * suffix - The string suffix to append to the end of the string.
+    * suffixes - Optional - Additional string suffixes that are valid terminators.
```

Comment:
Okay, I will retain additionalsuffix for now. Created METRON-847 to track the enhancement.